### PR TITLE
feat: add account backup warning test

### PR DESCRIPTION
**Pull request by Jayteekay**

### CODE REVIEW

File: src/test/java/dock/android/smoketests/appResetTrue/CreateAndBackupAccount.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        walletHomePage.clickNext();
+        walletHomePage.click(By.xpath("//*[contains(@text,'Copy to clipboard')]"));
+        walletHomePage.clickNext();
+        recoveryPhrase = driver.getClipboardText();
+        String index1Text = driver.findElement(By.xpath("/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.FrameLayout/android.view.ViewGroup/android.view.ViewGroup[2]/android.view.ViewGroup/android.view.ViewGroup[1]/android.widget.FrameLayout/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup[1]/android.widget.FrameLayout/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup[1]/android.widget.FrameLayout/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.widget.ScrollView/android.view.ViewGroup/android.view.ViewGroup[1]/android.widget.TextView[2]")).getText();
```
Comment (maycon-mello) [inline src/test/java/dock/android/smoketests/appResetTrue/CreateAndBackupAccount.java:48]: any chance it can break after layout changes?

Comment (maycon-mello) [reply src/test/java/dock/android/smoketests/appResetTrue/CreateAndBackupAccount.java:48]: it might not work on iOS right?

**Comment by maycon-mello** [comment]
I'm fine with the current implementation. But we might need to start using testIds instead of hardcoded locales, also avoid android specific selectors, otherwise, it won't work on iOS later on. 

We have a few UX updates planned for that wallet in next quarter, so we will need to refactor those tests

**Review by maycon-mello — APPROVED** [review]

**Comment by Jayteekay** [comment]
I agree. We need to use more testIds

**Comment by mike-parkhill** [comment]
Totally agree - testIds are far more stable

**Pull request merged**